arlo-client/src/components/DataEntry/BlockCheckbox.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Block = styled.label`
+  &.bp3-control.bp3-checkbox {
```

Comment:
do the Blueprint classes do much for you here? seems like their checkbox styles are pretty different from what we want it to look like. wondering if their buttons styles would be more useful

Reply:
@jonahkagan We do overwrite a lot of the styles, but not all, and when I tried to change it out it screwed things up royally, haha.

Reply:
@jonahkagan We do overwrite a lot, but it does a bunch behind the scenes that is necessary, evidently, since when I tried changing it out everything broke, haha.